solutions/src/main/java/ru/sberbank/school/task02/CurrencyCalc.java

```diff
@@ -0,0 +1,49 @@
+package ru.sberbank.school.task02;
+
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.Quote;
+import ru.sberbank.school.task02.util.Symbol;
+
+import java.math.BigDecimal;
+import java.util.Collections;
+import java.util.List;
+
+public class CurrencyCalc implements FxConversionService {
+
+    private ExternalQuotesService externalQuotesService;
+
+    public CurrencyCalc( ExternalQuotesService externalQuotesService) {
+        this.externalQuotesService = externalQuotesService;
+    }
+
+
+    @Override
+    public BigDecimal convert( ClientOperation operation, Symbol symbol, BigDecimal amount) {
+        if(amount == null || operation == null || symbol == null || amount.equals(BigDecimal.ZERO))
```

Comment:
&#39;if&#39; construct must use &#39;{}&#39;s.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1877567595>*